### ISSUE TITLE
A few questions

### ISSUE THREAD

**Issue by anoomolu** [murphy-slaw/worse-swimming#2]
Hiya! First off, thanks for making this mod - in all my time using mods to disable sprinting, I never realised how fast swimming was, so this will come in really handy ^^

I had a couple of questions, if you don't mind answering them:
- (I don't think it does atm but) does _Worse Swimming_ cover vertical movement too? It feels a little odd that sinking/rising feels faster than swimming, and could be a neat feature.
- With the default friction values, how fast is wading and (sprint) swimming in comparison to vanilla? Do you know if these would play nice with regular sprinting disabled, or would swimming still be too fast by comparison?
- Do you have any plans to touch on the breath meter system? Maybe out of scope for this mod but making doors waterloggable and other commonly used exploits would be very nice.

**Comment by murphy-slaw** [comment]
1. Currently it only covers horizontal movement. "Swimming upward" (jumping underwater) is a totally different code path than regular swimming and right now I don't mix in to it.
2. I honestly don't know how it compares, I just tweaked the default values until it "felt right".
3. I don't have any plans to mess with the breath system at the moment.

I originally made this mod for a modpack where I wanted to limit players' ability to swim across oceans. Eventually I ended up building a more complete stamina system instead (https://modrinth.com/mod/stamina-for-tweakers). That mod uses Additional Entity Attributes, which adds a swimming speed attribute that affects water movement on all three axes.
